frontend-instructor/src/components/footer/Footer.jsx

```
@@ -0,0 +1,72 @@
+          </Link>
+        </div>
+
+        <div className="flex gap-4 mt-4 md:mt-0">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            x="0px"
+            y="0px"
+            width="40"
+            height="40"
+            viewBox="0 0 48 48"
+          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)